servatrice/src/servatrice.cpp

```diff
@@ -934,6 +946,15 @@ int Servatrice::getNumberOfWebSocketPools() const
     return settingsCache->value("server/websocket_number_pools", 1).toInt();
 }
 
+QHostAddress Servatrice::getServerWebSocketHost() const
+{
+    QString host = settingsCache->value("server/websocket_host", "any").toString();
+    if (host == "any")
+        return QHostAddress::Any;
+    else
+        return QHostAddress(host);
```

Comment:
What happens if the string in the config file is not a valid address?

Reply:
The `QHostAddress` [ctor](https://doc.qt.io/qt-5/qhostaddress.html#QHostAddress-6) decides how to parse the string. From my tests it looks like it's dropping anything that's not numeric or a valid separator `.:`, and then parses the rest as an address.
Anyway, if an invalid address is detected, the call to listen() fails and servatrice aborts:
```
Starting server on host "0.0.0.1" port 4747
gameServer->listen(): Error: "The address is not available"
```

Reply:
That seems fine. The error message isn't great but it's at least fail fast